client/src/modules/cohort/model/actions.js

```diff
@@ -14,12 +14,11 @@ const fetchCohortsSuccess = data => ({
 });
 
 // Dispatchers
-export const createCohort = (name, description, ownerId) => dispatch =>
+export const createCohort = (name, description, adminId) => dispatch =>
   postData(`${ENDPOINT_URL}/cohorts/create`, {
     name,
     description,
-    ownerId,
-    adminIds: ownerId
+    adminIds: adminId
```

Comment:
shouldn't adminIds be an array? See in shoppingList controller also. Here string is saved... (Cohorts also...)

Reply:
it is saved as array, it's controlled by model

Reply:
<img width="958" alt="zrzut ekranu 2019-02-04 o 12 20 13" src="https://user-images.githubusercontent.com/27632432/52205415-44ae3e80-2877-11e9-8663-543ac504f4e7.png">